crypto/nrf_oberon/include/ocrypto_ecdsa_p256.h

```diff
@@ -13,55 +20,72 @@ extern "C" {
 #include <stddef.h>
 #include <stdint.h>
 
-
-// ecdsa P-256 signature key pair generation
-// pk[64]: the generated public key
-// sk[32]: the private key
-// returns 0 if sk is a legal secret key
+/**@brief ecdsa P-256 signature key pair generation
+ * @param[out]  pk  The generated public key
+ * @param       sk  The Private key
+ *
+ * @returns 0 if sk is a legal secret key
+ */
 int ocrypto_ecdsa_p256_public_key(
     uint8_t pk[64],
     const uint8_t sk[32]);
 
-// ecdsa P-256 signature generate
-// sig[64]: the generated signature
-// m[mlen]: the input message
-// sk[32]:  the secret key
-// ek[32]:  the session key
-// returns 0 if ek is a valid session key
+
+/**@brief ecdsa P-256 signature generate
+ *
+ * @param[out]  sig     The generated signature
+ * @param       m       The input message
+ * @param       mlen    Length of the message
+ * @param       sk      The secret key
+ * @param       ek      The session key
+ *
+ * @returns 0 if ek is a valid session key
```

Comment:
Is `@returns` valid doxygen?

Reply:
This and other return value dox in this file seem to omit a lot of info.

Reply:
\returns is valid doxygen, rest will be reported to content creators